app/attachments.js

```diff
@@ -167,3 +169,63 @@ exports.createAbsolutePathGetter = rootPath => relativePath => {
   }
   return normalized;
 };
+
+//      getTempPath :: AbsolutePath -> AbsolutePath
+exports.getTempPath = userDataPath => {
+  if (!isString(userDataPath)) {
+    throw new TypeError("'userDataPath' must be a string");
+  }
+  return path.join(userDataPath, TEMPPATH);
+};
+
+//      createWriterForTempAttachment :: AttachmentsPath -> ArrayBuffer -> Ext ->
+//                            { Filename , ArrayBuffer }
+exports.createWriterForTempAttachment = root => {
+  if (!isString(root)) {
+    throw new TypeError("'root' must be a path");
+  }
+
+  return async (arrayBuffer) => {
+    fse.ensureDirSync(root);
+
+    const buffer = Buffer.from( new Uint8Array(arrayBuffer) );
```

Comment:
Is this really what prettier does with the code? Have you run `yarn ready` on all of this?